pcsx2/DEV9/Win32/pcap_io_win32.cpp

```diff
@@ -77,12 +77,11 @@ bool load_pcap()
 		return true;
 
 	//Store old Search Dir
-	wchar_t* oldDllDir;
 	int len = GetDllDirectory(0, nullptr);
 	if (len == 0)
 		return false;
 
-	oldDllDir = new wchar_t[len];
+	wchar_t* oldDllDir = new wchar_t[len];
```

Comment:
This is unrelated but now that old Windowses are unsupported, this Set/Reset dance should really be replaced with AddDllDirectory + alternate search paths + RemoveDllDirectory. Might be worth looking into for a future PR.